src/Microsoft.Framework.Runtime/DependencyManagement/NuGetDependencyResolver.cs

```diff
@@ -218,40 +219,53 @@ public void Initialize(IEnumerable<LibraryDescription> packages, FrameworkName t
                                           IEnumerable<IPackagePathResolver> cacheResolvers,
                                           IPackage package)
         {
-            string expectedHash = null;
-
-            foreach (var resolver in cacheResolvers)
+            // REVIEW: without precomputed hash on disk, this part may hurt startup performance
```

Comment:
Yep, you're right. We need to keep them on disk for the package cache. The runtime however should never read anything from disk. Instead the lock file should have the sha.